### ISSUE TITLE
Allow to use the API for another user point of view

### ISSUE THREAD

**Issue by rgazelot** [Calendart/Office365Adapter#14]
We force the API to target the current user's data https://github.com/Calendart/Office365Adapter/blob/master/src/Office365Adapter.php#L41. But the API allow us to target another user's data using `/users/<id>`. I think that this should be available in our library.

**Comment by Taluu** [comment]
The thing is, the api (or how we are using it) already uses the oauth token of a current user. So I think that if we should ever se CalendArt with another user, we would need its oauth token, so we can be in the context of said user (and thus keep using `/me` endpoints)

**Comment by rgazelot** [comment]
Not really. You can use your access_token and request some informations about a user if he's a part of your group or team etc. If you have any granted access on him.
